wesnoth-sota.po

```diff
@@ -2973,41 +2995,41 @@ msgid ""
 "is quite weak and ill now, and breathing raggedly. I don’t know how much "
 "longer he can survive."
 msgstr ""
-"3 VI, 23 YW\n"
+"3. VI, roku 23 w. l.:\n"
 "\n"
-"Dostali jsem se přes vysoké pláně až do Arkan-thorie. Dále jsme se přesunuli "
-"hlouběji do hor, kde jsme konečně nalezli mága. To byl Ras-Rastabahnův plán "
-"a blízký cíl mu dodal trochu síly. Každopádně je ale velmi slabí a nemocný. "
-"I dýchání je pro něj náročné. Nevím, jak dlouho ještě může žít."
+"Přešli jsme náhorní plošiny a přebrodili Arkan-thorii. Pak jsme zamířili "
+"znovu do hor, kde jsme konečně nalezli mága. To byl Ras-Rastabahnův plán. "
+"Cíl v dohledu mu dodal trochu síly. Každopádně je ale velmi slabý a nemocný "
+"a chraplavě dýchá. Nevím, jak dlouho ještě může přežít."
 
 #. [side]: type=Elder Mage, id=Crelanu
 #: data/campaigns/Secrets_of_the_Ancients/scenarios/16_The_Mage.cfg:51
 #: data/campaigns/Secrets_of_the_Ancients/scenarios/16_The_Mage.cfg:54
 msgid "Crelanu"
-msgstr "Crelaner"
+msgstr "Krelan"
```

Comment:
v DM, LoW a SoF je všude shodně přeložen jako Krelanu, nechal bych to tak

slova končící na -u se [neskloňuji nebo skloňují podle vzoru pán](https://prirucka.ujc.cas.cz/?id=230)

jenže žádný z těch příkladů nekončí na -nu (Krelanu má tedy u konce -n- stejně jako pán -n na konci)

takže navrhuji svoje skloňování (zachovat Krelanu v 1. a 5. pádě a zbytek skloňovat podle vzoru pán):
1. Krelanu, 2. Krelana, 3. Krelanovi, 4. Krelana, 5. Krelanu, 6. Krelanovi, 7. Krelanem
```suggestion
msgstr "Krelanu"
```

Reply:
To jsem si neověřil, pardon. Jen jsem měl někde v paměti tu Krelanovu knihu.